### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by carnuare**
Add Docker Compose quickstart guide.

### CODE REVIEW

File: docs/deployment/docker-compose-local-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+Navigate to localhost:5100 to access the main page of Bluejay. A prompt like the following will pop up in your browser, where have to input the `USER_RENDER` and `PASS_RENDER` values specified in the `.env` file:
+![Login](../../static/img/deployment/docker-quickstart/login-render.png)
+![BJ](../../static/img/deployment/docker-quickstart/render-bluejay.png)
+
+Governify ecosystem with bluejay services should have been deployed in your machine.
+
+To stop the containers use:
+```
+docker-compose -f docker-bluejay/docker-compose-local.yaml --env-file .env down
```
Comment (motero2k) [inline docs/deployment/docker-compose-local-quickstart.md]: To stop the containers you should use "stop", not ''down''. ''down'' deletes the containers. That command is used when something has changed (env, docker-compose) and you need  to ''re-compose'' the containers using ''down'' and then ''up''.

You can explain both cases if you want.

**Review by motero2k — CHANGES_REQUESTED** [review]
Nice contibution. Please address the feedback in the comments on `docker-compose-local-quickstart`

File: docs/deployment/docker-compose-local-quickstart.md (hunk 1 above)
Comment (carnuare) [reply docs/deployment/docker-compose-local-quickstart.md]: Thank you for your feedback and clarification regarding the Docker Compose commands. You're absolutely correct, and I appreciate your input.

**Review by motero2k — APPROVED** [review]
Nice!

**Pull request merged**